legion/docs/source/configuration.md

```diff
@@ -212,10 +212,10 @@ dex:
         orgs:
         - name: legion-platform  # linked GitHub organizations
     staticPasswords:  # static hardcoded passwords for test
-    - email: "user@example.com"      
+    - email: pass@mail.com
       password: password
       hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W" # bcrypt hash of the string "password"
-      username: "user"
+      username: pass
```

Comment:
@kirillmakhonin - placeholder ? < put here user name >

Reply:
I guess `pass` is easier than `<put here user name>`